plugin/vcs/gitlab/gitlab.go

```diff
@@ -544,7 +548,7 @@ func (p *Provider) fetchPaginatedRepositoryActiveMemberList(ctx context.Context,
 		return nil, false, common.Errorf(common.NotFound, fmt.Errorf("failed to fetch repository members from URL %s", url))
 	} else if code >= 300 {
 		return nil, false,
-			fmt.Errorf("failed to read repository members from URL %s, status code: %d, body: %s",
+			fmt.Errorf("failed to fetch repository members from URL %s, status code: %d, body: %s",
```

Comment:
We use "fetch" in the error above, adjustment to be consistent, unless I'm misreading the intention.